system_modules/napsequence/src/sequenceplayer.h

```diff
@@ -69,6 +70,14 @@ namespace nap
          */
         bool load(const std::string& name, utility::ErrorState& errorState);
 
+        /**
+         * Load a sequence serialized as a binary blob
+         * @param buffer reference to binary data
+         * @param errorState contains error upon failure
+         * @return true on success
+         */
+        bool loadBinary(const std::vector<glm::uint8>& buffer, utility::ErrorState& errorState);
```

Comment:
Use `nap::uint8` instead of glm, from `nap/numeric.h`

Reply:
fixed